plugin/input/file/provider.go

```diff
@@ -413,12 +413,11 @@ func (jp *jobProvider) initJobOffset(operation offsetsOp, job *Job) {
 }
 
 // tryResumeJob job should be already locked and it'll be unlocked.
-func (jp *jobProvider) tryResumeJobAndUnlock(job *Job, filename string) bool {
+func (jp *jobProvider) tryResumeJobAndUnlock(job *Job, filename string) {
 	jp.logger.Debugf("job for %d:%s resumed", job.sourceID, job.filename)
 
 	if !job.isDone {
 		job.mu.Unlock()
-		return false
```

Comment:
Return here allows us not to rewrite filename and etc.

Reply:
My bad, I'll fix it